src/main/webapp/components/lists/list-create.js

```diff
@@ -0,0 +1,153 @@
+}
+
+const ListIconType = props => {
+  const { label } = props
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+      }}
+    >
+      <ListItemIcon>{iconList[label]}</ListItemIcon>
+      <ListItemText primary={label} />
+    </div>
+  )
+}
+
+export const ListCreatePopover = props => {
+  const { list, anchorEl, onClose, open, onSave } = props
+  return (
+    <Popover
+      open={open}
+      anchorEl={anchorEl}
+      onClose={onClose}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'center',
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'left',
+      }}
+    >
+      <Box style={{ margin: '10px' }}>
+        <ListCreate prevList={list} onClose={onClose} onSave={onSave} />
+      </Box>
+    </Popover>
+  )
+}
+
+export const ListCreate = props => {
+  const { prevList, onClose, onSave } = props
+  const isNewList = prevList === undefined
+
+  const [list, setList] = React.useState({
+    id: isNewList ? newUuid().replace(/-/g, '') : prevList.id,
+    list_icon: isNewList ? 'folder' : prevList.list_icon,
+    list_bookmarks: isNewList ? [] : prevList.list_bookmarks,
+    query: isNewList ? undefined : prevList.query,
+    title: isNewList ? 'Untitled List' : prevList.title,
+  })
+
+  const handleIconChange = event => {
+    setList({ ...list, list_icon: event.target.value })
+  }
+  const handleTitleChange = event => {
+    setList({ ...list, title: event.target.value })
+  }
+
+  return (
+    <FormControl fullWidth variant="outlined" margin="normal">
+      <TextField
+        style={{ marginBottom: '20px' }}
+        variant="outlined"
+        label="Title"
+        value={list.title}
+        onChange={handleTitleChange}
+      />
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'left',
+        }}
+      />
+      <RadioButton
+        label="Limit based on filter"
+        buttonText={[
+          { label: 'Yes', value: true },
+          { label: 'No', value: false },
+        ]}
+        value={list.useFilter === true}
+        onChange={e => setList({ ...list, useFilter: e })}
+      />
+      {/*TODO Add Filtering capablities*/}
+      <TextField
+        select
+        variant="outlined"
+        label="List Icon"
+        value={list.list_icon}
+        onChange={handleIconChange}
+        style={{ marginBottom: '20px' }}
+      >
+        {Object.keys(iconList).map(item => {
+          return (
+            <MenuItem key={item} value={item}>
+              <ListIconType label={item} />
+            </MenuItem>
+          )
+        })}
+      </TextField>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          width: '100%',
+          justifyContent: 'space-between',
+        }}
+      >
+        <Button
+          variant="outlined"
+          color="primary"
+          style={{ width: '49%' }}
```

Comment:
`50%`?

Reply:
I put it at 49% instead of 50% so that there was a little space between the buttons. When it was 50% there was no gap and it looked a little weird to me